pyproject.toml

```diff
@@ -143,17 +162,25 @@ ignore = [
     "ANN002", # Missing type annotation for `*args`
     "ANN101", # Missing type annotation for `self` in method
     "ANN102", # Missing type annotation for `cls` in classmethod
-    "D203", "D413", # we are not using too many blank lines
-    "D213", # we are using the first line for summary
-    "D406", "D407", # we are using google style docstring
+    "D203",
+    "D413",   # we are not using too many blank lines
+    "D213",   # we are using the first line for summary
+    "D406",
+    "D407",   # we are using google style docstring
 ]
 
 [tool.ruff.lint.per-file-ignores]
 "packages/*.py" = [
-    "D105", "D104", "D100", "D107", "D103", "D415", # missing docstring
-    "D101", "D102", # missing docstring in public class and method
+    "D105",
+    "D104",
+    "D100",
+    "D107",
+    "D103",
+    "D415", # missing docstring
+    "D101",
+    "D102", # missing docstring in public class and method
 ]
-"tests/*.py" = ["S101", "D"]
+"tests/**.py" = ["S101", "D", "ANN201"]
```

Comment:
applies ruff ignores over all python files in tests, adding ignores of "no return annotation" - happy to remove in favor of explicit `-> None`

see [ANN201](https://docs.astral.sh/ruff/rules/missing-return-type-undocumented-public-function/)